tests/php/src/Order/OrderEventListenerTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace WeDevs\Dokan\Test\Order;
+
+use WeDevs\Dokan\Order\OrderEventListener;
+use WeDevs\Dokan\Test\DokanTestCase;
+
+/**
+ * @group order-event-listener
+ *
+ * @note by default the withdrawal option is enable for completed order.
+ */
+class OrderEventListenerTest extends DokanTestCase {
+
+    private $order_event_listener;
+    private $seller_id;
+
+    public function setUp(): void {
+        parent::setUp();
+        $this->order_event_listener = new OrderEventListener();
+        $this->seller_id = $this->factory()->seller->create();
+    }
+
+    public function test_processing_order_with_trash_event() {
+        $order_id = $this->create_order( 'processing' );
```

Comment:
Why are you creating order manually where we have helper functions for creating single & multi-vendor order?

Reply:
I created a helper method to make multiple orders without repeating code.